### PR TITLE
Houssame working branch backup

### PR DESCRIPTION
**Pull request by HoussameH**
Clean TS regression fix, User-defined location added, Stratified TS fix for combo == [ ]  bug.
Plus various data missing warning, and documentation update

### CODE REVIEW

File: code/config.py [hunk 1]
```diff
@@ -244,22 +245,25 @@ def time_series_parameters(param):
     :rtype: dict
     """
     # Quantiles for time series
-    param["quantiles"] = [100, 90, 80, 70, 60, 50, 40, 30, 20, 10, 0]
+    param["quantiles"] = [90, 70, 60, 40]
```
Comment (kais-siala) [inline code/config.py]: can you undo this please

Comment (HoussameH) [reply code/config.py]: yes, I missed it when resetting the config.py

File: code/config.py [hunk 2]
```diff
@@ -246,20 +247,23 @@ def time_series_parameters(param):
...
-        "WindOn": {"all": []},  # dictionary of hub height combinations
+        "WindOn": {"all": [],},  # dictionary of hub height combinations
         "WindOff": {"80m": []},  # dictionary of hub height combinations
         "PV": {"all": []},  # list of orientation combinations
         "CSP": {"all": []},
     }
 
     # Stratified time series
-    param["modes"] = {"high": [100, 90, 80], "mid": [70, 60, 50, 40, 30], "low": [20, 10, 0], "all": param["quantiles"]}
+    param["modes"] = {"high": [90, 70], "mid": [60, 40], "low": [], "all": param["quantiles"]}
```
Comment (kais-siala) [inline code/config.py]: undo this

File: code/config.py [hunk 3]
```diff
@@ -246,20 +247,23 @@ def time_series_parameters(param):
...
     param["combo"] = {
         # dictionary of hub height and orientation combinations
-        "WindOn": {"2015": [60, 80, 100], "2030": [80, 100, 120], "2050": [100, 120, 140]},
+        "WindOn": {"100m": [100], "all":[],},
```
Comment (kais-siala) [inline code/config.py]: undo this

Comment (HoussameH) [reply code/config.py]: done

**Pull request merged**